components/ModalCard/index.jsx

```
@@ -63,9 +74,11 @@ class ModalCard extends Component {
 
 ModalCard.propTypes = {
   title: PropTypes.node,
-  footer: PropTypes.instanceOf(ModalCardFooter),
+  footer: PropTypes.node,
```

Not sure if this is the right approach, but it does silence this error: 
```
Failed prop type: Invalid prop `footer` of type `Object` supplied to `ModalCard`, expected instance of `ModalCardFooter`.
```